bedrock/newsletter/templates/newsletter/hacks.mozilla.org.html

```diff
@@ -6,80 +6,12 @@
 <html>
```

Comment:
I don't think this page is used at all now? At one time it was shown in an iframe on the Hacks blog but they don't have it in the new design (and I'm not sure this newsletter is still alive at all). I think we can probably just delete this (including the related JS/CSS bits).